backend/src/API/memberAPI.ts

```diff
@@ -74,6 +75,25 @@ export const deleteMember = async (email: string, user: IdolMember): Promise<voi
   );
 };
 
+export const notifyMember = async (
```

Comment:
This seems to be TeamEvent specific. Consider putting this in `teamEventAPI` instead?